tests/topotests/lib/pim.py

```diff
@@ -210,6 +226,101 @@ def _add_pim_rp_config(tgen, topo, input_dict, router, build, config_data_dict):
                     config_data_dict[dut].extend(config_data)
 
 
+def _add_pim6_rp_config(tgen, topo, input_dict, router, build, config_data_dict):
```

Comment:
Isn't this almost identical to `_add_pim_rp_config`? Maybe can be squeezed into a single one for pim/pim6?

Reply:
@ton31337 ,  I have added pim/pim6 config in single API. Thank you.